packages/table/src/common/grid.ts

```diff
@@ -309,7 +309,7 @@ export class Grid {
         return EXTREMA_NONE;
     }
 
-    public getRegionStyle(region: IRegion) {
+    public getRegionStyle(region: IRegion): CSSProperties {
```

Comment:
explicit return type to avoid `error TS4055: Return type of public method from exported class has or is using private name 'React.CSSProperties'.`